docs/tasks/20260415_0755_事業オフセットスライダー対応設計.md

```
@@ -0,0 +1,169 @@
+| 50〜（Top51〜） | **集約ノード** |
+
+- 支出先: ウィンドウ事業（ランク10〜49）からの流入額合計でランク付け → Top40を個別表示、残りは集約
+
+---
+
+## 実装設計
+
+### 新規状態・型
+
+| 状態 | 型 | デフォルト | URL param |
+|------|----|-----------|-----------|
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)